sdk/src/test/java/com/horizen/customtypes/CustomPublicKeyProposition.java

```diff
@@ -58,4 +62,15 @@ public String toString() {
     public byte[] pubKeyBytes() {
         return Arrays.copyOf(pubKeyBytes, PUBLIC_KEY_LENGTH);
     }
+
+    @Override
+    public ProvableCheckResult<CustomPrivateKey> canBeProvedBy(List<Secret> secrectList){
```

Comment:
Can we just let CustomPublicKeyProposition extend `AbstractSingleSecretProofOfKnowledgeProposition` to avoid code duplication?